src/ast/ast.ts

```diff
@@ -301,15 +300,8 @@ export type AstStatementRepeat = {
 export type AstStatementTry = {
     kind: "statement_try";
     statements: AstStatement[];
-    id: number;
-    loc: SrcInfo;
-};
-
-export type AstStatementTryCatch = {
-    kind: "statement_try_catch";
-    statements: AstStatement[];
-    catchName: AstId;
-    catchStatements: AstStatement[];
+    catchName?: AstId;
+    catchStatements?: AstStatement[];
```

Comment:
These two fields must be either defined or undefined simultaneously

Reply:
Done